public/wp-content/plugins/ccs-salesforce/includes/cli-commands.php

```diff
@@ -370,6 +385,10 @@ public function all()
         $this->updateFrameworkSearchIndex();
         $this->updateSupplierSearchIndex();
 
+        // reindex elasticsearch
+        $this->reindexFrameworkSearchIndex();
+        $this->reindexSupplierSearchIndex();
```

Comment:
```
 $this->frameworkSearchClient->reindex();
 ```

Reply:
This is actually in reference to other functions in this file from line 945 - 965.

As well as calling the reindex function they also have a WP_CLI success message which I thought would be useful to indicate when the reindexing is finished and is also in line with the style of the others i.e updateFrameworkIndex and updateSupplierIndex